packages/katello/katello/katello.cron

```diff
@@ -3,3 +3,6 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 
 # Script for initiating removal of orphaned content
 00 22 * * 0	root	foreman-rake katello:delete_orphaned_content RAILS_ENV=production >/dev/null 2>&1
+
+# Script to refresh all alternate content sources
+00 18 * * 0     root    foreman-rake katello:refresh_alternate_content_sources >/dev/null 2>&1
```

Comment:
The tabbing isn't lining up with the entry above

Reply:
Should we include RAILS_ENV=production like the one above?

Reply:
Yeah, let me fix it and push up

Reply:
Added production, and will push up. Here is my file, it looks like they are matching up:

```
SHELL=/bin/sh
PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin

# Script for initiating removal of orphaned content
00 22 * * 0     root    foreman-rake katello:delete_orphaned_content RAILS_ENV=production >/dev/null 2>&1

# Script to refresh all alternate content sources
00 18 * * 0     root    foreman-rake katello:refresh_alternate_content_sources RAILS_ENV=production >/dev/null 2>&1
```

Reply:
I think I see the issue, on the above line it has single tabs for white space, but yours are spaces.

Reply:
@ianballou updated to use tabs